numpy/core/fromnumeric.py

```diff
@@ -3436,17 +3436,18 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue):
     Notes
     -----
     The standard deviation is the square root of the average of the squared
-    deviations from the mean, i.e., ``std = sqrt(mean(abs(x - x.mean())**2))``.
-
-    The average squared deviation is normally calculated as
-    ``x.sum() / N``, where ``N = len(x)``.  If, however, `ddof` is specified,
-    the divisor ``N - ddof`` is used instead. In standard statistical
-    practice, ``ddof=1`` provides an unbiased estimator of the variance
-    of the infinite population. ``ddof=0`` provides a maximum likelihood
-    estimate of the variance for normally distributed variables. The
-    standard deviation computed in this function is the square root of
-    the estimated variance, so even with ``ddof=1``, it will not be an
-    unbiased estimate of the standard deviation per se.
+    deviations from the mean, i.e., ``std = sqrt(mean(a))``, where 
+    ``a = abs(x - x.mean())**2``.
+
+    The average squared deviation is typically calculated as ``a.sum() / N``,
+    where ``N = len(a)``. If, however, `ddof` is specified, the divisor
```

Comment:
Since `a` is used as the input argument, I think these lines should swap `a` and `x` for consistency. Then `a` will always mean "the original input" and `x` will mean "some intermediate answer"